src/pages/patientView/PatientViewPage.tsx

```diff
@@ -304,7 +304,7 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                                         sampleLabels={sampleManager.sampleLabels}
                                         sampleColors={sampleManager.sampleColors}
                                         sampleManager={sampleManager}
-                                        getWidth={()=>$(window).width()-140}
+                                        getWidth={()=>$(window).width() - ((patientViewPageStore.mutationData.result.length > 0) ? 140 : 40)}
```

Comment:
I just really don't like these exposed magic numbers, and the logic being duplicated out here and also inside it in the `<If>`.

Maybe the prop can be `getContainerWidth={()=>$(window).width()}` and inside it can use that to do this logic?